FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+status: draft
+---
+
+# Frequently Asked Questions (FAQs)
+
+## Why create a new foundation?
+
+There are already several foundations which host open source projects,
+and each has its own focus and requirements.
+The CF is a home for self-governing, code-centric open source projects.
+Each open source project has its own unique needs and goals,
+and not every foundation is the right fit for every project.
+
+There are many critical application frameworks and libraries within the open source ecosystem that are not currently part of a foundation.
+The CF is a new option for projects to consider when looking for a foundation to call home,
+helping to strengthen the open source ecosystem as a whole by ensuring more projects are sustainable in the long term.
+
+Information covering benefits of the CF are in the [README](README.md):
+
+- [Core Distinctions of the CF](README.md#core-distinctions-of-cf)
+- [CF Guiding Principles](README.md#our-guiding-principles)
+- [Focus and Vision](README.md#cfs-focus-and-vision)
+- [Community and Sponsorship](README.md#community-and-sponsorship)
+
+## Why not the Apache Software Foundation?
+
+The Apache Software Foundation (ASF) has a long standing tradition of being a home for open source projects of all types,
+and continues to be a great home for many.
+As part of a project joining the ASF, it is required to run on the infrastructure that is provided for projects, including mailing lists,
+and adhere to its processes.
+
+For projects with well established infrastructure and processes, it's preferable for these to remain unchanged when joining a foundation.
+Though the CF provides default processes for projects joining the foundation,
+they're not mandatory when projects already have processes in place which achieve the same outcomes.
```

Comment:
```suggestion
they're not mandatory.
```

Reply:
I'd be concerned taking out this statement implies a project could join without using the CF default processes and without any of its own processes

Reply:
I see. I didn't know about CS wanting to impose anything in this area?

My concern with the original wording is that if one doesn't have "processes in place" then the "default processes" of the CF become mandatory. IMO that looks overly stronghanded, as in other areas the "default processes" seem more like suggestions (a word I've often seen is "templates").

I would have expected a newborn project would have some flexibility - obviously they'd need _some_ processes but they could tailor things a little? Or what if they initially want to use the CF "templates" but then feel the need to evolve things slightly? It would seem unfair if they are not allowed, while other projects which joined with different processes would be allowed to retain such flexibility. cc/ @ebullient

Reply:
I only want to impose transparency to whatever it is that projects use to decide what they decide. I don't think we need to say "thou shalt have a process".

I agree with Sanne here. The requirement (for me) is that whatever it is is written down somehow (even if that comes down to the single maintainer making decisions based on what is best for the whole project given xyz factors).